src/metabase/util.clj

```diff
@@ -137,7 +137,7 @@
   "Is `s` a valid HTTP/HTTPS URL string?"
   ^Boolean [s]
   (let [validator (UrlValidator. (varargs String ["http" "https"])
-                                 (RegexValidator. "^\\p{Alnum}+([\\.|\\-]\\p{Alnum}+)*(:\\d*)?")
+                                 (RegexValidator. "^[\\p{Alnum}\\_]+([\\.|\\-][\\p{Alnum}\\_]+)*(:\\d*)?")
```

Comment:
looks great to me. I am wondering what that last `+` operator does since there's a * after it. Do you know which tests fail if that is removed? I don't see the purpose of it

Reply:
Ran them locally and got lots of failures. It seems to be doing quite a bit of work in there so carry on.